apps/modernization-ui/src/design-system/entry/single-value/SingleValueEntry.tsx

```diff
@@ -0,0 +1,93 @@
+import { AlertBanner } from 'alert';
+import classNames from 'classnames';
+import { Button } from 'components/button';
+import { Heading } from 'components/heading';
+import { ReactNode, useEffect } from 'react';
+import { Control, DefaultValues, FieldValues, FormProvider, useForm } from 'react-hook-form';
+import { useSelectValueEntryState } from './useSelectValueEntryState';
+import styles from './SingleValueEntry.module.scss';
+
+type Props<V extends FieldValues> = {
+    id?: string;
+    title: string;
+    defaultValues?: DefaultValues<V>;
+    errors?: string[];
+    onChange: (data: V) => void;
+    isDirty: (isDirty: boolean) => void;
+    formRenderer: (control: Control<V>) => ReactNode;
+    isFooter?: boolean;
+};
+
+export const SingleValueEntry = <V extends FieldValues>({
+    id,
+    title,
+    defaultValues,
+    errors,
+    onChange,
+    isDirty,
+    formRenderer,
+    isFooter
+}: Props<V>) => {
+    const form = useForm<V>({ mode: 'onBlur', defaultValues });
+    const { update, reset, state } = useSelectValueEntryState<V>();
+
+    useEffect(() => {
+        if (state.data) {
+            onChange(state.data);
+        }
+    }, [state.data]);
+
+    useEffect(() => {
+        isDirty(form.formState.isDirty);
+    }, [form.formState.isDirty]);
+
+    const handleReset = () => {
+        form.reset();
+        reset();
+    };
+
+    const handleSubmit = (value: V) => {
+        update(value);
+        form.reset(value, { keepDefaultValues: true });
+    };
+
+    return (
+        <section id={id} className={styles.input}>
+            <header>
+                <Heading level={2}>{title}</Heading>
+                <span>
+                    <span className="required"></span>All required fields for {title.toLowerCase()}
```

Comment:
Issue: I don't see in the design, did i misunderstand this? 
![Screenshot 2024-09-13 at 1 14 57 PM](https://github.com/user-attachments/assets/6e33155d-0757-4408-92ac-4add67ffcc23)